UnityProject/render.sh

```
@@ -0,0 +1,215 @@
+    echo "  1. Run from a desktop session (recommended)"
+    echo "  2. Use headless EGL (NVIDIA GPUs with proper drivers)"
+fi
+
+# Run Unity in batch mode
+# Note: We use batchmode but NOT nographics since we need GPU for video encoding
+# Note: We don't use -quit because the recording happens asynchronously after play mode starts
+#       BatchModeRecorder calls EditorApplication.Exit() when recording completes
+# -force-vulkan ensures GPU rendering on Linux (alternatively use -force-glcore for OpenGL)
+
+# Clear old log
+> unity_render.log
```

Shellcheck (0.11.0)

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)